plugin/src/main/java/com/cookpad/android/plugin/license/task/CheckLicenses.kt

```diff
@@ -182,36 +182,37 @@ object CheckLicenses {
         )
     }
 
-    @VisibleForTesting
```

Comment:
[Note] I removed this annotation because `resolveProjectDependencies` is exposed for `object UpdateLicenses`, which refers it publicly, not for testing 😅